packages/angular/package.json

```diff
@@ -26,10 +26,10 @@
   "author": "",
   "license": "Apache-2.0",
   "dependencies": {
-    "@dataconnect/default-connector": "workspace:*",
```

Comment:
oh i think this change solves the above error. thanks @maneesht

Reply:
I'm not quite sure it does. I'm testing it locally to double check

Reply:
Looks like it fixed it. Waiting on a new release

Reply:
thats amazing 🎉